offlinedatasci/miniCran.R

```diff
@@ -8,7 +8,7 @@ if (!require("miniCRAN")) {
 }
 types = c("source", "win.binary", "mac.binary")
 DC_pkgs = c("tidyverse", "RSQLite")
-pth = file.path(getwd(), "miniCRAN")
+pth = file.path(args, "miniCRAN")
```

Comment:
To clarify what's going on here I'd probably make this line `pth = file.path(ods_dir, "miniCRAN") and then immediately after Line 2 (where `args` is defined) add `ods_dir = args[1]`. We're only passing one argument, so this should be the same, but this will clarify that there is only one argument being used here here and that it is the ods_dir.